java/download-preplanned-map-area/src/main/res/layout-land/layout_offline_controls.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is there any reason to have a landscape layout as you're overriding config changes with: `android:configChanges="keyboardHidden|orientation|screenSize"` in the manifest.